bobcat/bb-core/src/main/java/com/cognifide/qa/bb/provider/selenium/webdriver/WebDriverType.java

```diff
@@ -155,10 +144,24 @@ public WebDriver create(Capabilities capabilities, Properties properties) {
       FirefoxBinary firefoxBinary = new FirefoxBinary(firefoxPath);
       firefoxBinary.setEnvironmentProperty("DISPLAY",
           properties.getProperty(ConfigKeys.WEBDRIVER_XVFB_ID));
-      return new FirefoxDriver(firefoxBinary, null, capabilities);
+      return handleWebDriverProxyCookie(properties,
+          new FirefoxDriver(firefoxBinary, null, capabilities));
     }
   };
 
+  private static WebDriver handleWebDriverProxyCookie(Properties properties, WebDriver driver) {
```

Comment:
method name is misleading when reading ` return handleWebDriverProxyCookie(properties, driver);`